src/providers/atcoder.cr

```diff
@@ -0,0 +1,31 @@
+class AtCoderStats < SiteStats
+  @@url_regex = Regex.new("https?://(www[.])?atcoder.jp/users/(?<username>\\w+)")
+  # https://github.com/kenkoooo/AtCoderProblems/blob/master/doc/api.md
+  @@kenko_api_url = "https://kenkoooo.com/atcoder/atcoder-api/v3/user/language_rank?user="
+
+  def self.fetch_lang_stats(username : String) : Hash(Langname, Int32)?
+    stats = Hash(Langname, Int32).new(0)
+
+    response = HTTP::Client.get(@@kenko_api_url + username)
```

Comment:
I generally prefer using the [URI class to specify query parameters](https://crystal-lang.org/api/1.8.2/URI.html#query%3AString%7CNil-instance-method) rather than string concatenation but I guess it doesn't really matter much here...